src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -1033,10 +1040,11 @@ li {
   background-color: rgb(55, 164, 71);
   color: rgb(255, 255, 255) !important;
   font-weight: bold;
-}
 
-::v-deep .theme--light.v-label--is-disabled {
-  color: white !important;
+  // make label visible on background color
+  ::v-deep .theme--light.v-label--is-disabled {
+    color: white !important;
+  }
```

Comment:
I moved this into the `highlightedRole` block as that's the only time we want this override. Otherwise it affects all disabled labels (including checkbox), which is incorrect.